solr/test-framework/src/java/org/apache/solr/SolrJettyTestBase.java

```diff
@@ -93,7 +99,11 @@ public static JettySolrRunner createAndStartJetty(
 
   public static JettySolrRunner createAndStartJetty(String solrHome, JettyConfig jettyConfig)
       throws Exception {
-    return createAndStartJetty(solrHome, new Properties(), jettyConfig);
+
```

Comment:
however the former code led to code that did a bunch more stuff than I see here, like using "collection1" as the configSet, for example.